drivers/spi/spi_context.h

```diff
@@ -245,6 +245,20 @@ bool spi_context_rx_on(struct spi_context *ctx)
 	return !!(ctx->rx_buf || ctx->rx_len);
 }
 
+static ALWAYS_INLINE
```

Comment:
don't make it always inline (this is only interesting for one-liners inline functions).